styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/DockerRunnerHandler.java

```diff
@@ -141,7 +141,8 @@ private String dockerRunnerStart(RunState state) throws IOException {
         dockerImage,
         ImmutableList.copyOf(command),
         executionDescription.dockerTerminationLogging(),
-        executionDescription.secret());
+        executionDescription.secret(),
+        state.data().triggerId());
```

Comment:
`triggerId` is probably deprecated as noted by comment. `TriggerUtil.triggerId(trigger)` feels better. @rouzwawi ?